docs/aaaCtl/Account-Dashboard.md

```diff
@@ -0,0 +1,34 @@
+# Account Dashboard
+
+To view the FullCtl Account Dashboard (aka aaaCtl), go to: account.fullctl.io, enter username (or email address)  and password and click Login. 
+   ![](img/userpass.png)
+
+The Account Dashboard has three tabs - Organization Dashboard, User Account and Billing. 
+   ![](img/threetabs.png)
+
+**Organization Dashboard** = access to FullCtl applications, Users and Organization API Keys. This information is managed by Organization Administrators. 
+
+**User Account** = edit personal account information, view Organization invitations, change preferences, access authentication, set personal API keys, and change password
+
+**Billing** = add billing contact/method, view service charges, see order history
+
+# Organization Account Settings
+The following options are in the  Organization Dashboard:
```

Comment:
There needs to be a blank line after this line, otherwise the bullet points won't render.